actors/account/tests/account_actor_test.rs

```diff
@@ -123,26 +109,50 @@ fn authenticate_message() {
     })
     .unwrap();
 
+    // Valid signature
     rt.expect_validate_caller_any();
     rt.expect_verify_signature(ExpectedVerifySig {
         sig: Signature::new_secp256k1(vec![]),
         signer: addr,
         plaintext: vec![],
         result: Ok(()),
     });
-    assert!(rt.call::<AccountActor>(3, params.clone()).unwrap().is_none());
 
+    assert!(rt
+        .call::<AccountActor>(Method::AuthenticateMessageExported as MethodNum, params.clone())
+        .unwrap()
+        .is_none());
+
+    rt.verify();
+
+    // Invalid signature
     rt.expect_validate_caller_any();
     rt.expect_verify_signature(ExpectedVerifySig {
         sig: Signature::new_secp256k1(vec![]),
         signer: addr,
         plaintext: vec![],
         result: Err(anyhow!("bad signature")),
     });
-    assert_eq!(
+    expect_abort_contains_message(
         ExitCode::USR_ILLEGAL_ARGUMENT,
-        rt.call::<AccountActor>(3, params).unwrap_err().exit_code()
+        "bad signature",
+        rt.call::<AccountActor>(Method::AuthenticateMessageExported as MethodNum, params.clone()),
     );
-
     rt.verify();
+
+    // Ok to call exported method number
```

Comment:
nit: since we deprecated `3` this is redundant with the Valid Signature case

Reply:
Quite right!